### ISSUE TITLE
Bug in configure script for core

### ISSUE THREAD

**Issue by jacobbond** [bertiniteam/b2#47]
I think there is a small bug somewhere in the configure script:

having compiled only Boost.System from among the Boost libraries, the test failed with:

```
checking whether the Boost::System library is available... yes
checking for exit in -lboost_system... yes
checking whether the Boost::Filesystem library is available... yes
configure: error: Could not link against boost_system !
```

So then I compiled Boost.Filesystem (so System and Filesystem had been compiled) and got:

```
checking whether the Boost::Filesystem library is available... yes
checking for exit in -lboost_filesystem... yes
checking whether the Boost::Chrono library is available... yes
checking whether the Boost::Regex library is available... yes
configure: error: Could not link against boost_filesystem !
```

Then I compiled Boost.Chrono and got:

```
checking whether the Boost::Chrono library is available... yes
checking for exit in -lboost_chrono... yes
checking whether the Boost::Regex library is available... yes
configure: error: Could not link against boost_chrono !
```

So I think the variable it calls to print "Could not link against" is one step behind.

**Comment by ofloveandhate** [comment]
just build all of boost at once.  i simply cannot recommend building boost one at a time for bertini2.

as far as this lag, i haven't reproduced it.  the code which finds boost is from the autoconf m4 archive.  i will try updating the m4 scripts if they have been updated.

**Comment by ofloveandhate** [comment]
this was fixed along the way in issue #48
